src/test/scala/edu/ie3/simona/test/common/input/PvInputTestData.scala

```diff
@@ -54,4 +54,175 @@ trait PvInputTestData
     Quantities.getQuantity(10, StandardUnits.S_RATED),
     0.95,
   )
+
+  protected val pvSouth1 = new PvInput(
+    UUID.fromString("7ac5bb15-36ee-42b0-902b-9cd520e241b3"),
+    "pv_south_1",
+    new OperatorInput(UUID.randomUUID(), "NO_OPERATOR"),
```

Comment:
```suggestion
OperatorInput.NO_OPERATOR_ASSIGNED,
```
there was a operator uuid assigned. But I would propose to use OperatorInput.NO_OPERATOR_ASSIGNED

Reply:
same for the others